istio/README.md

```diff
@@ -0,0 +1,19 @@
+# Istio manifest for Kubeflow
+
+The manifest is taken from OSS Istio pre-release
+[1.1-20190111-09-15](https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/release-1.1-20190111-09-15/).
+
+We need custom configuration as discussed in the
+[issue](https://github.com/kubeflow/kubeflow/issues/1909#issuecomment-438409215).
+
+Specifically, the things we changed are:
+
+1. The policy of configmap `istio-sidecar-injector` is changed from `enabled` to `disabled`.
+   According to this [table](https://github.com/istio/istio/issues/6476#issuecomment-399219937),
+   `policy=disabled` and `namespace label=enabled` is what we want.
```

Comment:
@lluunn Looks like this no longer applies:
https://github.com/kubeflow/manifests/blob/master/istio/istio-install/base/istio-noauth.yaml#L12596

Reply:
Sure, I'll retain the reference to the table.